scripts/gitlab/test_linux_stable.sh

```diff
@@ -5,4 +5,4 @@ set -e
 source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/../common/lib.sh"
 
 time cargo test --release --locked -p polkadot-node-core-dispute-coordinator --features disputes
-time cargo test --workspace --release --verbose --locked --features=runtime-benchmarks
+time cargo test --workspace --release --verbose --locked --features=runtime-benchmarks,runtime-metrics
```

Comment:
We probably want to test both variants, right?

Reply:
Yeah, but that'd probably double the pipeline duration.

Reply:
I wouldn't think so. We should definitely check that the release-able version is good at all times.

The `feature=disputes` will be removed soon, so this should be fine for the time being.

Reply:
Hmm, I think @andronik has a point